chat-client/src/client/messager.ts

```diff
@@ -9,6 +9,8 @@ import {
     InsertToCursorPositionParams,
     SendToPromptParams,
     TabIdReceivedParams,
+    CopyCodeToClipboardParams,
```

Comment:
Cannot those be chat-client internal types? Types in '@aws/chat-client-ui-types' should be the ones that clients (e.g. IDE Extension depends on).

Reply:
ah I think I confused the types for _all_ communication between the chat-client and the extension. Yes in that case I can make these internal instead